app/src/main/java/com/github/se/travelpouch/model/activity/Activity.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.se.travelpouch.model.activity
+
+import com.github.se.travelpouch.model.Location
+import com.google.firebase.Timestamp
+
```

Comment:
You can add documentation to explain what an activity is

Reply:
Done